core/src/main/java/io/questdb/log/LogRollingFileWriter.java

```diff
@@ -199,6 +228,10 @@ public void setSpinBeforeFlush(String spinBeforeFlush) {
         this.spinBeforeFlush = spinBeforeFlush;
     }
 
+    public void setSizeLimit(String sizeLimit){ this.sizeLimit = sizeLimit; }
```

Comment:
We don't use one-liners for methods or code blocks:
```suggestion
    public void setSizeLimit(String sizeLimit){ this.sizeLimit = sizeLimit; }
```

Same applies to `setLifeDuration`.